.github/workflows/release.yaml

```diff
@@ -0,0 +1,21 @@
+name: Latest Release
+
+on:
+  release:
+    types:
+      - created
+
+jobs:
+    test-job:
+      name: "Tag / Build / Push Container"
+      runs-on: ubuntu-latest
+      steps:
+      - name: Trigger Buildkite Pipeline
+        uses: buildkite/trigger-pipeline-action@v1.2.0
+        env:
+          BUILDKITE_API_ACCESS_TOKEN: ${{ secrets.Buildkite_Token }}
+          PIPELINE: ${{ github.repository }}
+          COMMIT: "HEAD"
+          BRANCH: "master"
+          MESSAGE: ":github: Triggered from a GitHub Action"
+          BUILD_ENV_VARS: "{\"BUILDKITE_TAG\": \"latest\"}"
```

Comment:
This needs to be a git tag I think.
The buildkite docker image build script will take care of the rest ... it has the logic for applying the "latest" tag.

Reply:
https://github.com/input-output-hk/latest_docker_dev

https://hub.docker.com/r/craigem/latest_docker_dev/tags